R/auditCommonFunctions.R

```diff
@@ -2019,17 +2019,97 @@
       
       startProgressbar(length(likelihoods))
       
-      for(i in 1:length(likelihoods)){
+      leftPlotError <- try({
+        
+        for(i in 1:length(likelihoods)){
+          if(options[["bayesianAnalysis"]]){
+            names <- c("Beta", "Gamma", "Beta-binomial")
+            ir <- base::switch(options[["IR"]], "High" = 1, "Medium" = 0.60, "Low" = 0.36, "Custom" = options[["irCustom"]])
+            cr <- base::switch(options[["CR"]], "High" = 1, "Medium" = 0.60, "Low" = 0.36, "Custom" = options[["crCustom"]])
+            
+            # Create a prior distribution that incorporates the existing information
+            prior <- jfa::auditPrior(materiality = performanceMateriality,
+                                     confidence = confidence,
+                                     expectedError = parentOptions[["expectedErrors"]],
+                                     likelihood =  likelihoods[i],
+                                     N = N,
+                                     ir = ir,
+                                     cr = cr,
+                                     method = options[["priorConstructionMethod"]],
+                                     sampleN = options[["sampleN"]],
+                                     sampleK = options[["sampleK"]],
+                                     factor = options[["factor"]],
+                                     pHplus = 1 - options[["pHmin"]],
+                                     pHmin = options[["pHmin"]])
+          } else {
+            names <- c("Binomial", "Poisson", "Hypergeometric")
+            prior <- FALSE
+          }
+          result <- jfa::planning(materiality = performanceMateriality,
+                                  confidence = confidence,
+                                  expectedError = parentOptions[["expectedErrors"]],
+                                  likelihood = likelihoods[i],
+                                  minPrecision = minPrecision,
+                                  N = N,
+                                  increase = options[["sampleSizeIncrease"]],
+                                  prior = prior)
+          n[i] <- result[["sampleSize"]]
+          k[i] <- result[["expectedSampleError"]]
+          progressbarTick()
+        }
+      })
+      
+      if(class(leftPlotError) != "try-error"){
```

Comment:
```suggestion
      if(!isTryError(leftPlotError)){
```
(function from `jaspBase`)